native-authentication/src/main/java/com/okta/nativeauthentication/form/Form.kt

```diff
@@ -15,17 +15,75 @@
  */
 package com.okta.nativeauthentication.form
 
+import com.okta.idx.kotlin.dto.IdxRemediation
 import kotlinx.coroutines.CoroutineScope
+import java.util.Objects
 
 class Form private constructor(
     val elements: List<Element>,
+    private val builtFromWithoutTransform: Builder,
+    private val builderToElementMap: Map<Element.Builder<*>, Element>,
 ) {
     internal data class Builder(
         val elements: MutableList<Element.Builder<*>> = mutableListOf(),
         val launchActions: MutableList<suspend CoroutineScope.() -> Unit> = mutableListOf(),
     ) {
-        fun build(): Form {
-            return Form(elements.map { it.build() })
+        fun build(formTransformers: List<FormTransformer>): Form {
+            val builtFromWithoutTransform = Builder(
```

Comment:
```suggestion
            val builtFormWithoutTransform = Builder(
```

Reply:
The was intentionally `builtFrom`. Basically marking the builder that was used to build the form.